frontend/src/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import { BrowserRouter } from 'react-router-dom';
+import { Auth0Provider } from "@auth0/auth0-react";
+
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Auth0Provider domain="dev-gr6iu32kohhfpye6.us.auth0.com"
```

Comment:
Add `domain` and `clientId` in `.env` both are sensitive details they should not be available on github.